CODE_OF_CONDUCT.md

```diff
@@ -25,6 +25,6 @@ Project maintainers have the right and responsibility to remove, edit, or reject
 
 This code of conduct applies both within project spaces and in public spaces when an individual is representing the project or its community.
 
-Instances of abusive, harassing, or otherwise unacceptable behavior in kubtest may be reported by contacting the kubtest Code of Conduct Committee via contact@kubeshop.io. 
+Instances of abusive, harassing, or otherwise unacceptable behavior in testkube may be reported by contacting the testkube Code of Conduct Committee via contact@kubeshop.io. 
```

Comment:
```suggestion
Instances of abusive, harassing, or otherwise unacceptable behavior in TestKube may be reported by contacting the TestKube Code of Conduct Committee via contact@kubeshop.io. 
```